images/win/scripts/Installers/Install-Ruby.ps1

```diff
@@ -17,4 +17,8 @@ $latestRubyBinPath2_5 = Get-ChildItem -Path $toolcacheRubyPath | Sort-Object {[S
 
 Add-MachinePathItem $latestRubyBinPath2_5
 $env:Path = Get-MachinePath
+
+# Update ruby gem to latest version
+gem update --system
```

Comment:
I believe we should report to docs the latest version of gem.

Reply:
Yes, sure. We have added docs generation.